components/download_images/src/main.py

```diff
@@ -88,21 +89,19 @@ def download_image_with_retry(
 class DownloadImagesComponent(DaskTransformComponent):
     """Component that downloads images based on URLs."""
 
-    def transform(
-            self,
-            dataframe: dd.DataFrame,
-            *,
-            timeout: int,
-            retries: int,
-            image_size: int,
-            resize_mode: str,
-            resize_only_if_bigger: bool,
-            min_image_size: int,
-            max_aspect_ratio: float,
-    ) -> dd.DataFrame:
-        """Function that downloads images from a list of URLs and executes filtering and resizing
+    def __init__(self,
+                 *_,
```

Comment:
what's the difference between `*_` and `*`?

Reply:
- `*` doesn't capture any arguments, it just indicates that all following arguments are keyword arguments
- `*args` captures all remaining positional arguments, but throws a linting error if you don't add it to the docstring args.
- `*_` is the same as `*args` but indicates that the captured arguments are not used. This is a [Python convention](https://stackoverflow.com/questions/5893163/what-is-the-purpose-of-the-single-underscore-variable-in-python) that linters recognize.